receiver/splunkenterprisereceiver/search_result.go

```diff
@@ -156,3 +157,17 @@ type DispatchArtifactContent struct {
 	StatusCacheSize    string `json:"cached_job_status_status_csv_size_mb"`
 	CacheTotalEntries  string `json:"cached_job_status_total_entries"`
 }
+
+// '/services/server/health/splunkd/details
+type HealthArtifacts struct {
```

Comment:
```suggestion
type healthArtifacts struct {
```
limit your exported API to the minimum to avoid dependencies down the road using your internal structs.